charts/ceph-csi-rbd/values.yaml

```diff
@@ -164,6 +164,10 @@ provisioner:
   # system-cluster-critical which is less priority than system-node-critical
   priorityClassName: system-cluster-critical
 
+  # enable hostnetwork for provisioner pod. default is false
+  # useful for deployments where the podNetwork has no access to ceph
+  enableHostNetwork: false
```

Comment:
add documentation for this in https://github.com/ceph/ceph-csi/blob/devel/charts/ceph-csi-rbd/README.md and  https://github.com/ceph/ceph-csi/blob/devel/charts/ceph-csi-cephfs/README.md

Reply:
Hm, i don't know where that additional commit came from. I'll take a look at it.

Reply:
Did as suggested. Way simpler..